snoop/management/commands/walkocr.py

```diff
@@ -1,13 +1,32 @@
+import sys
 from django.core.management.base import BaseCommand
-from django.conf import settings
+from ... import models
 from ...ocr import walk
 
 class Command(BaseCommand):
 
     help = "Ingest OCRed files"
 
     def add_arguments(self, parser):
-        parser.add_argument('tag')
+        parser.add_argument(
+            'collection_slug',
+            default=None,
```

Comment:
I don't think it makes sense to send `default=None` for either `collection_slug` or `tag`.